### PR TITLE
Drools 1180 Migrate QE tests upstream

### PR DESCRIPTION
**Pull request by baldimir**
This is fourth in a series of PRs that migrate some tests from QE regression test suite upstream (there will be more).

Migrated test classes that inherit from ParsersTest class in QE regression test suite.

### CODE REVIEW

File: drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/parser/ParserTests.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * 
+ * Directories: - to test DRL files use drl directory - to test DSL use dsl
+ * directory, both DSL and DSLR files must have the same name to be matched
+ * together - for files that should be part of smoke test use smoke directory
+ * (these does not necessarily have to be DRL) - in case of any other file type
+ * new folder can be added
+ * 
+ */
+@RunWith(Parameterized.class)
+public abstract class ParserTests {
```
Comment (psiroky) [inline drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/parser/ParserTests.java]: Is there a reason to use a plural in the class name?

Comment (baldimir) [inline drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/parser/ParserTests.java]: No, I missed that when migrating the test. You are right. I'll change that.

Comment (baldimir) [inline drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/parser/ParserTests.java]: Fixed.

**Comment by psiroky** [comment]
+1 to merge once the build is green

**Pull request merged**